source/aggregation-pipeline-builder.txt

```diff
@@ -73,8 +73,62 @@ Limitations
 Create an Aggregation Pipeline
 ------------------------------
 
+Aggregation pipelines can be created in 2 seperate modes:
```

Comment:
For numbers zero through nine, we should write out the name of the actual number. (See [Numbers](https://www.mongodb.com/docs/meta/style-guide/style/numbers/) in the style guide)